soccer/writers.py

```diff
@@ -135,49 +129,20 @@ def standings(self, league_table, league):
             # Define the upper and lower bounds for Champions League,
             # Europa League and Relegation places.
             # This is so we can highlight them appropriately.
-            cl_upper = LEAGUE_PROPERTIES[league]['cl'][0]
-            cl_lower = LEAGUE_PROPERTIES[league]['cl'][1]
-            el_upper = LEAGUE_PROPERTIES[league]['el'][0]
-            el_lower = LEAGUE_PROPERTIES[league]['el'][1]
-            rl_upper = LEAGUE_PROPERTIES[league]['rl'][0]
-            rl_lower = LEAGUE_PROPERTIES[league]['rl'][1]
+            cl_upper, cl_lower = LEAGUE_PROPERTIES[league]['cl']
+            el_upper, el_lower = LEAGUE_PROPERTIES[league]['el']
+            rl_upper, rl_lower = LEAGUE_PROPERTIES[league]['rl']
 
+            team_str = ("{position:7} {teamName:33} {playedGames:12}"
+               " {goalDifference:14} {points}").format(**team)
```

Comment:
The padding is a little off for `position` and `played`
![image](https://cloud.githubusercontent.com/assets/3130883/14327758/8abf5580-fbe8-11e5-99b8-6c339deb5a6e.png)